packages/story-editor/src/components/checklist/checks/firstPageAnimation.js

```diff
@@ -50,16 +71,26 @@ const FirstPageAnimation = () => {
   return isRendered && isChecklistMounted ? (
     <ChecklistCard
       title={title}
+      cta={
+        <DefaultCtaButton
+          aria-label={'Remove Animations'}
+          onClick={removeAnimations}
+        >
+          {'Remove Animations'}
+        </DefaultCtaButton>
+      }
       footer={
         <>
           <DefaultFooterText>{footer}</DefaultFooterText>
-          <Link
-            onClick={onClick}
-            href="https://wp.stories.google/docs/how-to/animations/"
-            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
-          >
-            {'Learn more'}
-          </Link>
+          <DefaultFooterText as="span">
+            <Link
+              onClick={onClick}
+              href="https://wp.stories.google/docs/how-to/animations/"
+              size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+            >
+              {'Learn more'}
```

Comment:
I know this was existing in the past, but might we need the internationalization here for text?